org.gnome.Totem.Videosite.YouTubeDl.yaml

```diff
@@ -0,0 +1,32 @@
+id: org.gnome.Totem.Videosite.YouTubeDl
+runtime: org.gnome.Totem
```

Comment:
```suggestion
runtime: org.gnome.Totem//stable
```

Reply:
that doesnt work, but builds on my system. is it a flathub problem?

Reply:
You have to use "runtime-version" to specify the base app version for it to work on flathub.

Reply:
When I added extensions support to totem, I tried to make sure not to need that. Does that mean that flathub extensions would need to be respun for every release bump? That does sound like a problem as Bernie mentioned :/

Reply:
I think `runtime-version` is equivalent of `//stable` notation but maybe I was wrong.

@hadess It's about flatpak branch, not runtime version. Unless it needs a rebuild every Totem release, that is.

Reply:
I got the cue from: https://flathub.org/builds/#/builders/32/builds/32332/steps/7/logs/err_text
It is equivalent in `flatpak-builder` (hence working locally)

I ran into that when doing the audio plugins (I have a program to generate manifests)

Reply:
Ha, so: `runtime-version: stable` should work, correct? Good, nothing that I'd need to change on my side ;)